modules/core/src/lib/attribute.js

```diff
@@ -218,6 +220,9 @@ export default class LayerAttribute extends Attribute {
           this.update({constant: false, value: buffer});
           state.needsRedraw = true;
         }
+      } else {
+        this.update(buffer);
```

Comment:
Getting a little lost here. What is the type of the buffer in this case? Why are we adding this, typed arrays should be fine on workers? 

Maybe a comment could help?